File: common/src/main/java/dev/obscuria/tooltips/ObscureTooltips.java
package dev.obscuria.tooltips;

import dev.obscuria.tooltips.registry.TooltipsRegistries;
import net.minecraft.resources.ResourceLocation;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public interface ObscureTooltips {

    String MOD_ID = "obscure_tooltips";
    String MOD_NAME = "Obscure Tooltips";
    Logger LOGGER = LoggerFactory.getLogger(MOD_NAME);

    static ResourceLocation key(String name) {
        return new ResourceLocation(MOD_ID, name);
    }

    static void init() {
        TooltipsRegistries.init();
    }
}

File: common/src/main/kotlin/dev/obscuria/tooltips/ObscureTooltips.kt
package dev.obscuria.tooltips

import dev.obscuria.tooltips.registry.TooltipsRegistries
import net.minecraft.resources.ResourceLocation
import org.slf4j.Logger
import org.slf4j.LoggerFactory

object ObscureTooltips {

    const val MODID = "obscure_tooltips"
    const val DISPLAY_NAME = "Obscure Tooltips"
    val LOGGER: Logger = LoggerFactory.getLogger(DISPLAY_NAME)

    fun key(name: String): ResourceLocation {
        return ResourceLocation(MODID, name)
    }

    fun init() {
        TooltipsRegistries.init()
    }
}